### ISSUE TITLE
Handle Gitlab as a git provider

### ISSUE THREAD

**Issue by spoukke** [padok-team/git-volume-reloader#2]

**Comment by viktoriavikulya** [comment]
Worth checking if this reproduces with a fresh install — some package managers cache broken versions.

**Comment by kimberly-WY** [comment]
One thing worth trying: setting the timeout at the transport layer rather than the request level — avoids the race condition.

**Comment by kimberly-WY** [comment]
One thing worth trying: setting the timeout at the transport layer rather than the request level — avoids the race condition.

**Comment by Deniwn22** [comment]
Worth checking if this reproduces with a fresh install — some package managers cache broken versions.

**Comment by viktoriavikulya** [comment]
Worth checking if this reproduces with a fresh install — some package managers cache broken versions.

**Comment by Katomoto** [comment]
Confirmed reproducible on Node 20 + macOS. Interestingly it doesn't happen if you set NODE_OPTIONS=--max-old-space-size=512.

**Comment by relda88** [comment]
One thing worth trying: setting the timeout at the transport layer rather than the request level — avoids the race condition.

**Comment by viktoriavikulya** [comment]
Good catch. This is related to the RFC change in how trailing slashes are normalized. PR incoming.
